### ISSUE TITLE
configure.ac: `neurosim/config.h.in` not found

### ISSUE THREAD

**Issue by sanjayankur31** [INCF/libneurosim#18]
Hello,

while trying to build the latest commit for version 1.2.0, on running `autogen.sh`, I get this:

```
 ./autogen.sh 
aclocal -I /usr/share/aclocal
libtoolize --force --copy --automake --ltdl
autoheader
autoconf
automake --foreign --copy --add-missing
configure.ac:247: error: required file 'neurosim/config.h.in' not found

```

**Comment by jougs** [comment]
@mdjurfeldt, does this have to do with you switching the ConnGen interface to version 2?

**Comment by mdjurfeldt** [comment]
@jougs I'll have a look at this today.

**Comment by mdjurfeldt** [comment]
Should be fixed now in commit 24657e8

**Comment by sanjayankur31** [comment]
Thanks, I'll update the [NeuroFedora package](https://src.fedoraproject.org/rpms/libneurosim) to the latest commit when I get a chance next week.
